code/game/objects/items/storage/backpack.dm

```diff
@@ -197,6 +199,9 @@
 	desc = "A trendy looking satchel."
 	icon_state = "satchel-norm"
 	species_exception = list(/datum/species/angel) //satchels can be equipped since they are on the side, not back
+	block_open_while_equipped = 1

```

Comment:
Wait shouldn't this be `block_open_while_equipped = 0` so it _can_ be opened?

Reply:
Yup, artifact from when I had the var named can_open_while_equipped, but decided to flip the name because it makes the check easier to read.